.github/workflows/ci.yml

```diff
@@ -259,3 +259,27 @@ jobs:
       - name: Notebook check
         run: |
           check/pytest_.py --notebook
+
+  docs:
+    name: Build docs
+    runs-on: ubuntu-latest
+    timeout-minutes: 7
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
```

Comment:
Should this check use 3.11 like the server-side equivalent, or 3.8 like the other checks in this file?

Reply:
`3.11`